hedera-mirror-rest/tokens.js

```diff
@@ -26,6 +26,13 @@ const EntityId = require('./entityId');
 const utils = require('./utils');
 const {InvalidArgumentError} = require('./errors/invalidArgumentError');
 const {NotFoundError} = require('./errors/notFoundError');
+const NftModel = require('./models/nftModel');
+const NftTransferModel = require('./models/nftTransferModel');
+const TransactionModel = require('./models/transactionModel');
+const TransactionTypeService = require('./services/transactionTypesService');
+const TokenService = require('./services/tokenService');
+const NftViewModel = require('./viewmodels/nftViewModel');
```

Comment:
Singular package names.

```suggestion
const NftViewModel = require('./viewmodel/nftViewModel');
```

Reply:
Not my preference, but  consistency is the name of the game so I'm okay with matching the Java project.